bittensor/_neuron/text/advanced_server/run.py

```diff
@@ -187,7 +186,7 @@ def time_check():
     dataset = bittensor.dataset(config=config)
 
     # load our old model
-    if config.neuron.no_restart != True:
+    if not config.neuron.no_restart :
```

Comment:
Does using `not` instead of `!=` really make a huge difference? I'm just curious.

Reply:
nooo, it was just for aligning with the rest of the code

Reply:
and seems a little bit easier to read